recipe/meta.yaml

```diff
@@ -1,91 +1,122 @@
-{% set version = "1.9.1" %}
+{% set version = "1.9.6" %}
+{% set name = "geoviews" %}
 
 package:
-  name: geoviews-core
+  name: {{ name }}-split
   version: {{ version }}
 
 source:
-  url: https://github.com/holoviz/geoviews/archive/v{{ version }}.tar.gz
-  sha256: 458bbe40b4bbf7bba5ef7b99d89a28618c024102545bc03687b6659368b335ed
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2ab66626fb57434ede7670ed8b2b2e8331919dbec78edb4d1d1646dfa0c73e7
 
 build:
   number: 0
+  #skip s390x as lack of nodejs and cartopy dependencies
+  skip: True  # [py<37 or s390x]
 
 outputs:
-  - name: geoviews-core
-    script: build_base.sh
+  - name: {{ name }}-core
+    script: build_base.sh # [not win]
+    script: build_base.bat # [win]
     build:
       noarch: python
       entry_points:
         - geoviews = geoviews.__main__:main
     requirements:
       run_constrained:
-        - geoviews {{ version }}
+        - {{ name }} {{ version }}
       host:
-        - python >=3.6
+        - python
         - pip
-        - param >=1.9.0
-        # These are also needed at build time and declared in the pyproject.toml
-        - pyct >=0.4.4
-        - bokeh >=2.3.0,<2.4.0
-        - nodejs >=10.13.0
+        - setuptools
         - wheel
-        - cartopy >=0.18.0
-        - setuptools >=30.3.0
+        - packaging
+        - param 1.9.0
+        - pyct 0.5.0
+        - bokeh 2.4.3
+        - nodejs 16.13.1
+        - cartopy 0.21.1
+        - pyviz_comms 0.7.2
       run:
-        - python >=3.6
-        - bokeh >=2.3.0,<2.4.0
+        - python
+        - bokeh >=2.4.0,<2.5.0
         - cartopy >=0.18.0
         - holoviews >=1.14.2
-        - numpy  >=1.0
+        - numpy >=1.0
```

Comment:
I don't see any pinning in the upstream:
https://github.com/holoviz/geoviews/blob/v1.9.6/setup.py#L101

Reply:
in the CF https://github.com/conda-forge/geoviews-feedstock/blob/8eea41b379f6636247bcdf48e809b97617753ab6/recipe/meta.yaml#L40

Reply:
Can use {{ numpy }} in this case I think

Reply:
I think the source of truth should be the upstream, not CF. CF is just for reference. I might be wrong.

Reply:
I think that any package ver < 1.0 is immature,  but I will remove that constrain